docs/markdown/BUILD.bazel

```diff
@@ -99,9 +103,18 @@ nodejs_binary(
 nodejs_binary(
     name = "markdown",
     data = [
+        "//bazel/browsers/chromium:chromium-for-generation",
         "@npm//jsdom",
+        "@npm//mermaid",
+        "@npm//playwright-core",
     ],
     entry_point = "//docs/markdown:guides.mjs",
+    env = {
+        "CHROME_BIN": "$(CHROMIUM)",
```

Comment:
This has implications I think because: Chromium on Darwin has spaces in its files. Remember all the reports in Slack where macOS team memebers get errors because runfiles in bazel don't support spaces.

Reply:
Should I switch to using firefox for it you think?

Reply:
Mhmm. Interesting idea. Might be worth doing yeah 😄

Reply:
So firefox we don't have windows support.  So I am not sure what our options really are :D

Reply:
In my testing I did not have any issues on os x

Reply:
Okay, we might need to revisit this though if comes up. See all the reports for integration tests in #fw-caretaker